src/clarity-angular/emphasis/alert/alerts.ts

```diff
@@ -13,10 +13,10 @@ import {MultiAlertService} from "./providers/multi-alert-service";
     providers: [MultiAlertService],
     host: {
         "[class.alerts]": "true",
-        "[class.alert-danger]": "this.currentAlertType == 'danger'",
-        "[class.alert-info]": "this.currentAlertType == 'info'",
-        "[class.alert-success]": "this.currentAlertType == 'success'",
-        "[class.alert-warning]": "this.currentAlertType == 'warning'"
+        "[class.alert-danger]": "this.currentAlertType == 'danger' || this.currentAlertType == 'alert-danger'",
```

Comment:
Can we put a comment that these are deprecated so we can find it later? Same on the other PR now that I think of it.

`/** @deprecated */`

Reply:
Yep. Will do. Thanks!